sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-processing-steps/validate-disbursement-base.ts

```diff
@@ -39,6 +39,14 @@ export abstract class ValidateDisbursementBase {
       log.info(`Student MSFAA associated with the disbursement is not signed.`);
       validationResults.push(ECertFailedValidation.MSFAANotSigned);
     }
+    // MSFAA invalid.
+    if (
+      eCertDisbursement.disbursement.msfaaNumber.cancelledDate ||
+      !eCertDisbursement.disbursement.msfaaNumber.dateSigned
+    ) {
+      log.info(`Student MSFAA associated with the disbursement is not valid.`);
+      validationResults.push(ECertFailedValidation.MSFAANotValid);
+    }
```

Comment:
If the assumption from the below comment is right, this code should be reverted.
https://github.com/bcgov/SIMS/pull/3298/files#r1621449475